composer.json

```diff
@@ -1,43 +1,45 @@
 {
     "name": "guardrdistro/guardr-project",
-    "description": "Project template for Guardr Drupal 8 projects with composer",
+    "description": "Project template for Guardr Drupal 9 projects with composer",
     "type": "project",
     "license": "GPL-2.0-or-later",
+    "homepage": "https://www.drupal.org/project/guardr",
     "authors": [
         {
             "name": "Mark Shropshire",
             "homepage": "https://www.drupal.org/u/shrop",
             "role": "Maintainer"
         }
     ],
-    "repositories": [
-        {
-          "type": "vcs",
-          "url": "https://git.drupalcode.org/project/guardr.git"
+    "repositories": {
+        "guardr": {
+            "type": "vcs",
+            "url": "https://git.drupalcode.org/project/guardr.git"
         },
-        {
-          "type": "vcs",
-          "url": "https://git.drupalcode.org/project/guardr_core.git"
+        "guardr_core": {
+            "type": "vcs",
+            "url": "https://git.drupalcode.org/project/guardr_core.git"
         },
-        {
+        "drupal": {
             "type": "composer",
             "url": "https://packages.drupal.org/8"
         }
-    ],
+    },
     "require": {
-        "php": ">=7.3",
-        "composer/installers": "^1.2",
-        "cweagans/composer-patches": "^1.7.0",
+        "php": ">=7.4 || ^8",
+        "composer/installers": "^1.9",
+        "cweagans/composer-patches": "^1.7",
         "drupal/console": "^1.9",
-        "drupal/core": "^9.0",
-        "drupal/core-composer-scaffold": "^9.0",
-        "drupal/guardr": "^2.0.0",
-        "drush/drush": "^10.3",
-        "vlucas/phpdotenv": "^4.0",
-        "webflo/drupal-finder": "^1.0.0"
+        "drupal/core-composer-scaffold": "^9.4",
+        "drupal/core-project-message": "^9.4",
+        "drupal/core-recommended": "^9.4",
+        "drupal/guardr": "dev-9.x-test",
```

Comment:
Isn't "drupal/guardr": "^2.0.0" correct here?

Reply:
Hoping this can be merged at some point and I'll update this again to use 2.x branch.
https://git.drupalcode.org/project/guardr/-/merge_requests/2/commits